packages/flutter/lib/src/material/dialog.dart

```diff
@@ -242,6 +266,19 @@ class AlertDialog extends StatelessWidget {
   /// from the [actions].
   final List<Widget> actions;
 
+  /// {@macro flutter.material.dialog.backgroundColor}
+  final Color backgroundColor;
+
+  /// The z-coordinate of this [Dialog].
+  ///
+  /// If this is `null`, [DialogTheme.elevation] will be used, if
```

Comment:
As before:
If null then [DialogTheme.elevation] is used, and if that's null then the dialog's elevation is 24.0.

Reply:
Fixed in macro.